onsite.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: single-page-md
+title: APIO 2020 Onsite
+key: onsite
+---
+
+### Registration for Onsite Contest
+* There will be only 10 that are invited and 6 participants for each country. The committee will accept the confirmation of the country that wants to do it onsite in the First Come First Serve basis.
+* To register your country, please refer first to the general [registration page](registration).
+* In the confirmation phase, the leader needs to state their confirmation to participate in onsite contest.
+* In the registration phase, the leader needs to state the participants that will participate in the onsite contest.
+
+### Onsite Contest Information
+* The APIO 2020 onsite have the same rule as the [online contest format](rules#online-contest-format), with several differences.
+* The participants of each invited country will take the contest in Jakarta, Indonesia.
+* The contest will be held on Sunday, 10 May 2020 (UTC+7 08.00) for 5 hours.
+* Besides the contest day, there will be a time for excursion and visiting popular places in Jakarta.
+* There are no differences between the onsite participants and the online participants. The winners will be decided equally as if the onsite participants are the online participants.
+* If onsite participants [win medals](rules#medals-distribution), they will be awarded physical medals in the closing ceremony / awarding night.
+
+
+### Onsite Contest Schedule
+
+Draft schedule, subject to change! For the general schedule, check [the schedule page](schedule).
```

Comment:
`s/Draft schedule, subject to change/Plesae note that the following is a draft schedule and subject to change.`